website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,151 @@
+
+**2.** Determine if ACLs are enabled by running the following CLI command.
+
+```shell
+consul info | grep "acl ="
+````
+
+If you receive output like either of the following then ACLs are enabled:
+
+```text
+Error querying agent: Unexpected response code: 403 (Permission denied)
+```
+
+```text
+        acl = enabled
+
+```
+
+Select the correct tab below based on whether ACLs are enabled or disabled.
+
+<Tabs>
+<Tab heading="ACLs Enabled">
+
+**3.** Upgrade all server agents to the latest 1.8.x or 1.9.x release.
+
+**4.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
+
+**5.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to the latest 1.8.x or 1.9.x release.
+
+**6.** Pre-configure the server agents with a license.
+
+This can either be set with the `license_path` configuration item, the 
+`CONSUL_LICENSE_PATH` environment variable or the `CONSUL_LICENSE` 
+environment variable. See the [licensing documentation](/docs/enterprise#licensing) 
+for more information about how to configure the license.
+
+**7.** Upgrade all server agents to v1.10.0
+
+**8.** Upgrade all client agents to v1.10.0
+
+**9.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to v1.10.0
+
+</Tab>
+<Tab heading="ACLs Disabled">
+
+~> We do not recommend running in production with ACLs disabled. An alternative upgrade
+path involves first enabling ACLs by following [this tutorial](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production)
+and then proceeding with the instructions within the "ACLs Enabled" tab.
+
+**3.** Upgrade all server agents to the latest 1.8.x or 1.9.x release.
+
+**4.** Pre-configure the client agents with a license.
+
+This can either be set with the `license_path` configuration item, the 
+`CONSUL_LICENSE_PATH` environment variable or the `CONSUL_LICENSE` 
+environment variable. See the [licensing documentation](/docs/enterprise#licensing) 
+for more information about how to configure the license. Note that while
+Consul Enterprise 1.8.13 and 1.9.7 have the ability to load the license
+from these configurations, it is intended to only be used during upgrades.
+Licenses loaded this way are not online reloadable. It is therefore important
+to promptly proceed with the upgrade and not leave agents in the intermediate state.
+
+**5.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
+
+**6.** Pre-configure the snapshot agents with a license. This is the same process that
+was executed for the servers in step 4.
+
+**7.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to the latest 1.8.x or 1.9.x release.
+
+**8.** Pre-configure the server agents with a license. This is the same process that
+was executed for the servers in step 4.
+
+**9.** Upgrade all server agents to v1.10.0.
+
+**10.** Upgrade all client agents to v1.10.0.
+
+**11.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to v1.10.0.
+
+</Tab>
```

Comment:
Tentative Kubernetes Docs
```suggestion
</Tab>
<Tab heading="Kubernetes">
  
**3.** Upgrade to the latest Consul helm chart v0.32.0+.

**4.** Update the value of `global.image` in the values file to the latest 1.8.x or 1.9.x image.
Additionally, ensure that the Kuberenetes secret with the license is correctly specified in the
values `server.enterpriseLicense.secretName` and `server.enterpriseLicense.secretKey`.

**5.** Upgrade the cluster.
  
Follow the steps in the [Kubernetes Upgrade Guide](docs/k8s/upgrade#upgrade-consul-on-kubernetes)
to then upgrade the cluster. Ensure you check the steps to upgrade the [servers](docs/k8s/upgrade#upgrading-consul-servers)
and the [clients](https://www.consul.io/docs/k8s/upgrade#upgrading-consul-clients).

**6.** Update the value of `global.image` in the values file to the 1.10.0 image.

**7.** Upgrade the cluster.

Follow the steps in the [Kubernetes Upgrade Guide](docs/k8s/upgrade#upgrade-consul-on-kubernetes)
to then upgrade the cluster. Ensure you check the steps to upgrade the [servers](docs/k8s/upgrade#upgrading-consul-servers)
and the [clients](https://www.consul.io/docs/k8s/upgrade#upgrading-consul-clients).
  
</Tab>
```

Reply:
https://github.com/hashicorp/consul/pull/10449

Reply:
What about if only clients are in k8s?